README.md

```diff
@@ -8,14 +8,14 @@ This project will be awesome!
 
 TL;DR: Convenient one-liner for Ubuntu:
 ``` bash
-$ sudo apt install -y cmake libboost-all-dev libopencv-dev libeigen3-dev libpng16-dev
+$ sudo apt install -y cmake libboost-all-dev libopencv-dev libeigen3-dev libpng-dev
 ```
 
 - CMake: `$ sudo apt install cmake`
 - Boost.Test: `$ sudo apt install libboost-all-dev`
 - [OpenCV](https://docs.opencv.org/trunk/d7/d9f/tutorial_linux_install.html): `$ sudo apt install libopencv-dev`
 - Eigen: `$ sudo apt install libeigen3-dev`
-- We need to read PNG files: `$ sudo apt install libpng16-dev`
+- We need to read PNG files: `$ sudo apt install libpng-dev`
```

Comment:
Hmm, looking at it again: I just checked apt-get to make sure I don't break anything, I do. So I'll roll it back. 

That's a change I did one or two weeks ago, OpenCV also uses libpng, and somewhere it is only listed to be `libpng-dev`. So I adjusted our dependency to be compatible (only necessary in the readme). In general, it can be annoying if A uses Bv1 and C, while C uses Bv2. I tried to avoid this, but since it would cause more harm then help, rolling back ...

Note: On travis it's actually running `libpng12-dev`, since 1.6 isn't available. This tells me, that we don't actually use any unstable interfaces, so it doesn't matter too much in the end.